scripts/startup_graph.py

```diff
@@ -0,0 +1,113 @@
+import json
+import pandas as pd
+import plotly.express as px
+import numpy as np
+import argparse
+
+
+def readable_time(nanoseconds):
+    if nanoseconds >= 1e9:
+        return f"{nanoseconds / 1e9:.2f} s"
+    elif nanoseconds >= 1e6:
+        return f"{nanoseconds / 1e6:.2f} ms"
+    elif nanoseconds >= 1e3:
+        return f"{nanoseconds / 1e3:.2f} μs"
+    else:
+        return f"{nanoseconds} ns"
+
+
+parser = argparse.ArgumentParser(description="Generate Gantt chart of startup times of MicroShift services.")
```

Comment:
Not to confuse with systemd services.
```suggestion
parser = argparse.ArgumentParser(description="Generate a Gantt chart of MicroShift internal services startup times .")
```

Reply:
done